packages/block-library/src/section/editor.scss

```diff
@@ -0,0 +1,55 @@
+$block-name: "section";
+
+/**
+ * Section: All Alignment Settings
+ */
+.wp-block[data-type="core/#{$block-name}"] {
+	padding-left: 42px;
```

Comment:
@getdave One thing I came across while working on the Twenty Nineteen styles: these padding rules shouldn't apply to tiny screens. Otherwise, the section block becomes much narrower than necessary on mobile.

Reply:
Good spot. Should be resolved as of https://github.com/WordPress/gutenberg/pull/13964/commits/9007e58a69399507916af8b9f169362390611896